src/Core/Checkout/Promotion/Exception/PriceNotFoundException.php

```diff
@@ -7,6 +7,9 @@
 use Shopware\Core\Framework\ShopwareHttpException;
 use Symfony\Component\HttpFoundation\Response;
 
+/**
+ * @deprecated tag:v6.8.0 - reason:remove-exception - Will be removed, use PromotionException::priceNotFound() instead
+ */
 #[Package('checkout')]
 class PriceNotFoundException extends ShopwareHttpException
```

Comment:
For an easier transition, these exceptions should start with the domain exception and inherit the error code